src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -277,17 +278,20 @@
 
 (defn message-content-wrapper
   "Author, userpic and delivery wrapper"
-  [{:keys [last-in-group? identicon from in-popover? timestamp-str
-           deleted-for-me? deleted-for-me-undoable-till pinned]
+  [{:keys [last-in-group? identicon from in-popover? timestamp-str deleted?
+           deleted-for-me? deleted-undoable-till deleted-for-me-undoable-till
+           pinned]
     :as   message} content {:keys [modal close-modal]}]
   (let [response-to (:response-to (:content message))]
-    (if deleted-for-me?
+    (if (or deleted? deleted-for-me?)
       [system-message/system-message
        {:type             :deleted
-        :label            :message-deleted-for-you
+        :label            (if deleted? :message-deleted :message-deleted-for-you)
         :timestamp-str    timestamp-str
         :non-pressable?   true
-        :animate-landing? (if deleted-for-me-undoable-till true false)}]
+        :animate-landing? (if (or deleted-undoable-till deleted-for-me-undoable-till)
+                            true
+                            false)}]
```

Comment:
Deleted and deleted-for-me use the same `system-message` type, only difference is the label